github/ci/prow-deploy/kustom/overlays/workloads-production/resources/shared-images-controller.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: shared-images-controller
+  namespace: kubevirt-prow
+  labels:
+    name: shared-images-controller
+spec:
+  selector:
+    matchLabels:
+      name: shared-images-controller
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        name: shared-images-controller
+    spec:
+      nodeSelector:
+        type: bare-metal-external
+      terminationGracePeriodSeconds: 1
+      containers:
+      - name: shared-images-controller
+        image: quay.io/kubevirtci/shared-images-controller:v20221208-a669d6f
+        command: [ "/usr/local/bin/runner.sh", "/shared-images-controller"]
+        resources:
+          requests:
+            memory: 2Gi
+        securityContext:
+          privileged: true
```

Comment:
Does it need to be privileged?

Reply:
It requires a running podman socket to manage the images - for now I think we require it to be privileged in order to start podman in the container. 
It would be cool to see if we could move this and the e2e jobs to not require privileged but I think we would need to deploy the fuse-device-plugin to try rootless without privileged - https://www.redhat.com/sysadmin/podman-inside-kubernetes

Reply:
Got it. I think we can do without but its not blocking issue